crates/driver/src/boundary/settlement.rs

```diff
@@ -440,3 +450,9 @@ impl AllowanceManaging for AllowanceManager {
             .collect())
     }
 }
+
+fn to_array<const N: usize>(data: &[u8]) -> [u8; N] {
```

Comment:
This is in the `std` with [`&[u8]; TryInto<[u8; N]`](https://doc.rust-lang.org/std/primitive.array.html#impl-TryFrom%3C%26%5BT%5D%3E-for-%5BT%3B%20N%5D).